internal/v2/api/.ogen.yml

```diff
@@ -8,4 +8,4 @@ generator:
       - "client/request/validation"
     disable_all: true
   filters:
-    path_regex: ".*(?:v4/integrations|v1/groups|v2/access-flows|v1/access-scopes|v3/connectors|v2/users).*"
\ No newline at end of file
+    path_regex: ".*(?:v4/integrations|v1/groups|v2/access-flows|v1/access-scopes|v3/connectors|v2/users|v2/bundles).*"
```

Comment:
this is the change that caused all the generation changes... don't think you need to review them... all generated code.
(Other than the README.md).